xyz2mol.py

```diff
@@ -768,7 +765,9 @@ def main():
     parser.add_argument('-o', '--output-format',
         action="store",
         type=str,
-        help="Output format [smiles,sdf] (default=sdf)")
+        default="sdf",
```

Comment:
Not sure what the intended default behaviour is since the previous options where pointing in opposite directions... SDF or SMILES?